hoot-core/src/main/cpp/hoot/core/cmd/ConvertCmd.cpp

```diff
@@ -127,6 +132,50 @@ class ConvertCmd : public BaseCommand
     return 0;
   }
 
+  boost::shared_ptr<ElementCriterion> _getStreamingCriterion(const QStringList ops)
+  {
+    QStringList criterionNames;
+    Factory& factory = Factory::getInstance();
+    for (int i = 0; i < ops.size(); i++)
+    {
+      const QString opName = ops[i];
+      LOG_VART(opName);
+      if (!opName.trimmed().isEmpty())
+      {
+        if (factory.hasBase<ElementCriterion>(opName.toStdString()))
+        {
+          criterionNames.append(opName);
+        }
```

Comment:
Wait, I'm not sure about that...  In the case where I was converting geonames to OSM, I didn't need to specify a POI criterion op, b/c the input was already all POI's.  But I wanted streaming I/O, b/c I couldn't fit the input in memory.